src/test/scala/io/qbeast/spark/delta/QbeastSnapshotTest.scala

```diff
@@ -99,7 +99,7 @@ class QbeastSnapshotTest extends QbeastIntegrationTestSpec {
     }
 
   it should "respect the (0.0, 1.0] range" in withQbeastContextSparkAndTmpDir { (spark, tmpDir) =>
-    withOTreeAlgorithm { oTreeAlgorithm =>
+    withOTreeAlgorithm { _ =>
```

Comment:
if we are not using it, why don't we remove the withOTreeAlgorithm?

Reply:
Sure!